Source/WebKit/WebProcess/WebProcess.cpp

```diff
@@ -614,6 +614,7 @@ void WebProcess::initializeWebProcess(WebProcessCreationParameters&& parameters,
         RetainPtr<CFDataRef> auditData = adoptCF(CFDataCreate(nullptr, (const UInt8*)&*auditToken, sizeof(*auditToken)));
         Inspector::RemoteInspector::singleton().setParentProcessInformation(WebCore::presentingApplicationPID(), auditData);
     }
+    Inspector::RemoteInspector::singleton().notifyCanConnectToWebInspector();
```

Comment:
im pretty sure that Web Inspector will actually attempt to connect to `com.apple.webinspector` in the UIProcess as part of support for PSON, so is this actually necessary for Web Inspector to function?

Reply:
At least in the Service Worker case, I believe we need the connection to `com.apple.webinspector`.